src/app/views/rolodex_view.js

```diff
@@ -0,0 +1,90 @@
+import Backbone from 'backbone';
+import $ from 'jquery';
+import _ from 'underscore';
+import Contact from '../models/contact';
+import ContactView from './contact_view';
+
+const RolodexView = Backbone.View.extend({
+  initialize: function(params) {
+
+    this.template = params.template;
+    this.modal = $('#contact-details');
```

Comment:
I would suggest passing this in as a parameter instead of hardcoding it.

Alternatively, you should at least use `this.$` instead of raw jQuery to select elements.